docs/install-windows.md

```diff
@@ -0,0 +1,9 @@
+!!! warning
+    This page is a work-in-progress,
+    and will be updated.
```

Comment:
Let's try and avoid this kind of warning. It's the equivalent of a `FIXME` in code. If the content isn't sufficient, then we should either remove the page or improve the content.